java/test/jmri/configurexml/LoadAndStoreTestBase.java

```diff
@@ -160,29 +169,40 @@ static void loadFile(File inFile) throws Exception {
         new jmri.jmrit.catalog.configurexml.DefaultCatalogTreeManagerXml().readCatalogTrees();
     }
 
-    static File storeFile(File inFile) throws Exception {
+    // removed "static" here so we can access saveType field
+    private File storeFile(File inFile) throws Exception {
         String name = inFile.getName();
         FileUtil.createDirectory(FileUtil.getUserFilesPath() + "temp");
         File outFile = new File(FileUtil.getUserFilesPath() + "temp/" + name);
-        InstanceManager.getDefault(ConfigureManager.class).storeConfig(outFile);
+
+        // "Prefs", "UserPrefs", "Config", "User" or "All"
+        if (saveType.equals("Prefs")) {
+            InstanceManager.getDefault(ConfigureManager.class).storePrefs(outFile);
+        } else if (saveType.equals("UserPrefs")) {
+            InstanceManager.getDefault(ConfigureManager.class).storeUserPrefs(outFile);
+        } else if (saveType.equals("Config")) {
+            InstanceManager.getDefault(ConfigureManager.class).storeConfig(outFile);
+        } else if (saveType.equals("User")) {
+            InstanceManager.getDefault(ConfigureManager.class).storeUser(outFile);
+        } else if (saveType.equals("All")) {
+            InstanceManager.getDefault(ConfigureManager.class).storeAll(outFile);
+        } else {
+            log.error("Unknown save type {}.", saveType);
+        }
+
         return outFile;
     }
 
     @Test
     public void loadLoadStoreFileCheck() throws Exception {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
 
         log.debug("Start check file " + this.file.getCanonicalPath());
 
-        loadFile(this.file);
-        loadFile(this.file);
-
-        String name = this.file.getName();
-
-        // store file
-        FileUtil.createDirectory(FileUtil.getUserFilesPath() + "temp");
-        File outFile = new File(FileUtil.getUserFilesPath() + "temp/" + name);
-        InstanceManager.getDefault(ConfigureManager.class).storeConfig(outFile);
+        loadFile(this.file);    // <== I don't know why this was being called twice
+        ///loadFile(this.file); // <== If I comment out the 2nd what will break?
```

Comment:
As @bobjacobsen noted, removing this second loadFile() call invalidates the tests here. If this breaks the tests for you, do not make LELoadAndStoreTest inherit from this class, but implement LELoadAndStoreTest such that it can rely heavily on the static methods in this class (since I think this is the only method that causes problems for LELoadAndStoreTest).